src/sync/submitters/uniqueKeysSubmitter.ts

```diff
@@ -3,36 +3,18 @@ import { ISet, setToArray } from '../../utils/lang/sets';
 import { submitterFactory } from './submitter';
 import { UniqueKeysPayloadCs, UniqueKeysPayloadSs } from './types';
 
-/** 
- * Invert keys for feature to features for key
- */
-function invertUniqueKeys(uniqueKeys: { [featureName: string]: ISet<string> }): { [key: string]: string[] } {
-  const featureNames = Object.keys(uniqueKeys);
-  const inverted: { [key: string]: string[] } = {};
-  for (let i = 0; i < featureNames.length; i++) {
-    const featureName = featureNames[i];
-    const featureKeys = setToArray(uniqueKeys[featureName]);
-    for (let j = 0; j< featureKeys.length; j++) {
-      const featureKey = featureKeys[j];
-      if (!inverted[featureKey]) inverted[featureKey] = []; 
-      inverted[featureKey].push(featureName);
-    }
-  }
-  return inverted;
-}
-
 /**
  * Converts `uniqueKeys` data from cache into request payload for CS.
  */
 export function fromUniqueKeysCollectorCs(uniqueKeys: { [featureName: string]: ISet<string> }): UniqueKeysPayloadCs {
   const payload = [];
-  const featuresPerKey = invertUniqueKeys(uniqueKeys);
-  const keys = Object.keys(featuresPerKey);
-  for (let k = 0; k < keys.length; k++) {
-    const key = keys[k];
+  const featureKeys = Object.keys(uniqueKeys);
```

Comment:
I think this change has broken the logic:
For example, if it gets `{ f1: new Set(['k1']) }`, it returns `` instead of `{ keys: [{ k: 'k1', fs: ['f1'] }] }`

If the logic is broken, lets fix it, but also we should add some test to avoid breaking it again.

```
test('fromUniqueKeysCollectorCs', () => {
  expect(fromUniqueKeysCollectorCs({ f1: new Set(['k1']) })).toBe({ keys: [{ k: 'k1', fs: ['f1'] }] });
  // Some more asserts, multiple feature names and keys, etc
});
```